.github/workflows/ci.yaml

```
@@ -319,6 +329,14 @@ jobs:
           repository: securecodebox/scanner-gitleaks
           path: ./scanners/gitleaks/scanner/
           tags: "v6.1.2,latest"
+      - uses: docker/build-push-action@v1
+        name: "Build & Push Git-Repo-Scanner Image"
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: securecodebox/scanner-git-repo-scanner
+          path: ./scanners/git-repo-scanner/scanner/
+          tags: "v0.1,latest"
```

See versioning notes in Chart.yaml

```suggestion
          tags: "latest"
          tag_with_ref: true
```